inc/modules/export/epub/class-epub.php

```diff
@@ -585,7 +600,7 @@ protected function preProcessBookContents( $book_contents ) {
 	 *
 	 * @return string
 	 */
-	protected function preProcessPostContent( $content, $type = '', $post_id = 0 ) {
+	protected function preProcessPostContent( string $content, string $type = '', int $post_id = 0 ): string {
```

Comment:
@fdalcin I'm not sure if we are ready to use typed properties and return values because those I remember are only PHP 7.4 and currently we are still supporting PHP 7.3